src/Components/CodeEditor/CodeEditor.js

```diff
@@ -5,12 +5,14 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import messages from '../../Messages';
 import { useIntl } from 'react-intl';
+import { formatDate } from '../SigTable/helper';
 
 const CodeEditor = ({ code, language, isReadOnly, isDownloadEnabled, isCopyEnabled, height = '400px' }) => {
     const intl = useIntl();
 
     return <PfCodeEditor
         isDownloadEnabled={isDownloadEnabled}
+        downloadFileName={`malware-detection_signature--${formatDate(Date.now(), true)}`}
```

Comment:
The second argument, the `true` looks redundant. `formatDate` only has the `date` to provide.

Reply:
You are right, nice catch